Driver_LED.lib

```diff
@@ -989,9 +989,7 @@ F0 "U" -550 750 50 H V C CNN
 F1 "LT3755xMSE" 350 750 50 H V C CNN
 F2 "Package_SO:MSOP-16-1EP_3x4mm_P0.5mm_EP1.65x2.85mm" 1000 -750 50 H I C CNN
 F3 "" -50 -1100 50 H I C CNN
-ALIAS LT3755xMSE-2
-ALIAS LT3756xMSE
-ALIAS LT3756xMSE-2
+ALIAS LT3755xMSE-2 LT3756xMSE LT3756xMSE-2
```

Comment:
There is something weird going on here. 
The previous change to this file was done in https://github.com/KiCad/kicad-symbols/pull/2257, which I also reviewed. I assume it was with a different KiCad version.
Your commit cleared that up.